source/declarations.tex

```diff
@@ -4809,7 +4809,7 @@
 \item
 no virtual functions\iref{class.virtual}, and
 \item
-no virtual, private, or protected base classes\iref{class.mi}.
+no virtual, direct private, or direct protected base classes\iref{class.mi}.
```

Comment:
Perhaps better to split this into two bullets?

Reply:
Or "or base classes" can be added to the previous bullet, and "virtual" removed from this one?

Reply:
> Or "or base classes" can be added to the previous bullet, and "virtual" removed from this one?

Sounds good.